### PR TITLE
Doi update

### PR DESCRIPTION
**Pull request by AdamTheisen**
- [ ] PEP8 Standards or use of linter

### CODE REVIEW

**Review by mgrover1 — APPROVED** [review]
I like this refactoring! Thank you @AdamTheisen

File: act/discovery/get_armfiles.py [hunk 1]
```diff
@@ -198,22 +198,8 @@ def get_arm_doi(datastream, startdate, enddate):
...
-    inst_class = response['instrument_class_code']
-
     # Get the DOI information
-    doi_url = 'https://adc.arm.gov/citationservice/citation/inst-class?id=' + inst_class + '&citationType=apa'
-    doi_url += '&site=' + site
-    doi_url += '&dataLevel=' + level
+    doi_url = 'https://adc.arm.gov/citationservice/citation/datastream?id=' + datastream + '&citationType=apa'
     doi_url += '&startDate=' + startdate
     doi_url += '&endDate=' + enddate
     doi = requests.get(url=doi_url)
```
Comment (mgrover1) [inline act/discovery/get_armfiles.py:205]: Can you add a try except value error block here?

**Comment by zssherman** [comment]
Looks like the Value Error test isn't raising a valueerror so it's failing

**Comment by AdamTheisen** [comment]
Datastream option is returning a different result.  Updating the tests and adding in the try for the get...  Testing and then will upload

**Review by mgrover1 — APPROVED** [review]

**Pull request merged**